Keas.Mvc/ClientApp/components/Equipment/EquipmentContainer.tsx

```diff
@@ -195,12 +195,16 @@ export default class EquipmentContainer extends React.Component<IProps, IState>
         person.id
       }&date=${date}`;
 
+      if(!equipment.assignment)
+      {
+        // don't count as assigning unless this is a new one
+        assigned = true;
```

Comment:
@laholstege seems a bit odd, why are we setting assigned = true if there is no equipment assignment? Is it just the variable naming?

Reply:
@srkirkland you're right, it's kind of confusing. the variable is used to determine if the function for updating counts should be called. so it should only be true if this is a new assignment.

Reply:
@laholstege so it's more like a shouldUpdate or hasChanges or similar?  I have to make a big deal about variables because if I'm confused now i'll really be confused reading this again in 3 years